package.json

```diff
@@ -1,10 +1,44 @@
 {
   "name": "client-side-validations",
-  "private": true,
+  "version": "12.0.0beta7",
   "scripts": {
-    "lint": "coffeelint -- coffeescript/rails.validations.coffee"
+    "build": "rollup -c",
+    "dev": "rollup -c -w",
+    "test": "node test/test.js",
+    "pretest": "npm run build"
   },
   "devDependencies": {
-    "coffeelint": "^2.0.7"
+    "babel-core": "^6.26.3",
+    "babel-jest": "^22.4.3",
+    "babel-preset-env": "^1.6.0",
+    "coffeelint": "^2.0.7",
+    "jest": "^22.4.3",
+    "jquery": "^3.0",
+    "rollup": "^0.58.2",
+    "rollup-plugin-babel": "^2.7.1",
+    "rollup-plugin-commonjs": "^8.0.2",
+    "rollup-plugin-node-resolve": "^3.0.0",
+    "standard": "^11.0.1"
+  },
+  "dependencies": {},
+  "peerDependencies": {
+    "jquery": "^3.0"
+  },
+  "main": "dist/client-side-validations.cjs.js",
+  "module": "dist/client-side-validations.esm.js",
+  "browser": "dist/client-side-validations.umd.js",
+  "esnext": "dist/client-side-validations.esnext.js",
+  "files": [
+    "dist"
+  ],
+  "standard": {
+    "ignore": [
+      "/test/javascript/public",
+      "/test/javascript/views",
+      "/dist"
+    ]
+  },
+  "jest": {
+    "transform": { "^.+\\.jsx?$": "babel-jest" }
```

Comment:
Could you please clarify this?

Why `jsx`?

Reply:
That's the default transform regex from their example, but actually I ended up not using jest since the existing tests are already fast and provide good coverage. I'll remove jest from the dependancies.